ballerina/soap.bal

```diff
@@ -59,9 +59,10 @@ public isolated client class Client {
     #
     # + body - SOAP request body as an `XML` or `mime:Entity[]` to work with SOAP attachments
     # + action - SOAP action as a `string`
+    # + headers - SOAP headers as a `map<string|string[]>`
     # + return - If successful, returns the response. Else, returns an error
-    remote function sendReceive(xml|mime:Entity[] body, string action) returns xml|mime:Entity[]|Error {
-        return sendReceive(self.soapVersion, body, self.soapClient, action);
+    remote function sendReceive(xml|mime:Entity[] body, string action, map<string|string[]>? headers = {}) returns xml|mime:Entity[]|Error {
```

Comment:
```suggestion
    remote function sendReceive(xml|mime:Entity[] body, string action, map<string|string[]> headers = {}) returns xml|mime:Entity[]|Error {
```
Since we define headers as `{}`, do we need it be `nil` as well?

Reply:
Fixes in [b467557](https://github.com/ballerina-platform/module-ballerina-soap/pull/96/commits/b4675571ad5237522e22cd3d0b8fc38305291d2d)